R/plotSpectra-PTM.R

```diff
@@ -352,147 +350,142 @@ plotSpectraPTM <- function(x, deltaMz = TRUE, ppm = 20,
 
 ##' Annotated sequence fragments split view
 ##'
-##' Code based on plotting tool [MMS2Plot](https://doi.org/10.1002/pmic.202000061)
-##'
 ##' @param x Spectra object
 ##'
 ##' @param peptide_sequence `character(1L)` The identified peptide sequence
 ##'
 ##' @param mzs mz values of peaks
 ##'
-##' @param name intensity values of peaks
+##' @param ints the intensity values of peaks
```

Comment:
Specify the type of the input such as `numeric(1)` or `numeric()`, ... same of all arguments from this function.